### PR TITLE
refactor : use utils methods

### PR DESCRIPTION
**Pull request by LinMoskovitch**

### CODE REVIEW

File: Notify/Notify.Functions/Notify.Functions/NotifyFunctions/Login/CheckUserExists.cs [hunk 1]
```diff
@@ -35,13 +36,11 @@ public static class CheckUserExists
 
             try
             {
-                collection = AzureDatabaseClient.Instance.GetCollection<BsonDocument>(Constants.DATABASE_NOTIFY_MTA,
-                    Constants.COLLECTION_USER);
+                collection = Utils.MongoUtils.GetCollection(Constants.COLLECTION_USER);
                 log.LogInformation(
                     $"Got reference to {Constants.COLLECTION_USER} collection on {Constants.DATABASE_NOTIFY_MTA} database");
```
Comment (OfirMatasas) [inline Notify/Notify.Functions/Notify.Functions/NotifyFunctions/Login/CheckUserExists.cs:41]: This log can be removed.

File: Notify/Notify.Functions/Notify.Functions/NotifyFunctions/Login/Login.cs [hunk 2]
```diff
@@ -36,13 +38,12 @@ public static class Login
 
             try
             {
-                collection = AzureDatabaseClient.Instance.GetCollection<BsonDocument>(Constants.DATABASE_NOTIFY_MTA,
-                    Constants.COLLECTION_USER);
+                collection = MongoUtils.GetCollection(Constants.COLLECTION_USER);
                 log.LogInformation(
```
Comment (OfirMatasas) [inline Notify/Notify.Functions/Notify.Functions/NotifyFunctions/Login/Login.cs:42]: This log can be removed.

File: Notify/Notify.Functions/Notify.Functions/NotifyFunctions/Login/RegisterUser.cs [hunk 3]
```diff
@@ -35,15 +36,14 @@ public static class UserRegistration
 
             try
             {
-                collection = AzureDatabaseClient.Instance.GetCollection<BsonDocument>(Constants.DATABASE_NOTIFY_MTA,
-                    Constants.COLLECTION_USER);
+                collection = MongoUtils.GetCollection(Constants.COLLECTION_USER);
+                
                 log.LogInformation(
```
Comment (OfirMatasas) [inline Notify/Notify.Functions/Notify.Functions/NotifyFunctions/Login/RegisterUser.cs:41]: This log can be removed.

File: Notify/Notify.Functions/Notify.Functions/NotifyFunctions/Notification/DeleteNotification.cs [hunk 4]
```diff
@@ -37,9 +38,7 @@ public static class DeleteNotification
             
             log.LogInformation($"Got reference to {Constants.COLLECTION_NOTIFICATION} collection on {Constants.DATABASE_NOTIFY_MTA} database");
```
Comment (OfirMatasas) [inline Notify/Notify.Functions/Notify.Functions/NotifyFunctions/Notification/DeleteNotification.cs:39]: This log can be removed, and forgot the collection above this log.

**Review by OfirMatasas — CHANGES_REQUESTED** [review]

File: Notify/Notify.Functions/Notify.Functions/NotifyFunctions/Notification/DeleteNotification.cs (hunk 4 above)
Comment (LinMoskovitch) [reply Notify/Notify.Functions/Notify.Functions/NotifyFunctions/Notification/DeleteNotification.cs:39]: Fixed

File: Notify/Notify.Functions/Notify.Functions/NotifyFunctions/Notification/DeleteNotification.cs [hunk 5]
```diff
@@ -34,9 +35,7 @@ public static class DeleteNotification
             log.LogInformation($"Got client's HTTP request to delete notifications");
 
             database = AzureDatabaseClient.Instance.GetDatabase(Constants.DATABASE_NOTIFY_MTA);
```
Comment (OfirMatasas) [inline Notify/Notify.Functions/Notify.Functions/NotifyFunctions/Notification/DeleteNotification.cs:37]: Can be removed as well.

**Review by OfirMatasas — APPROVED** [review]
LGTM, just don't forget to remove the call for the database object as I mentioned.

**Pull request merged**